_posts/en/newsletters/2023-07-26-newsletter.md

```diff
@@ -0,0 +1,240 @@
+      described in [Newsletter #226][news226 jamming] and setting the
+      stage for a later implementation of low upfront fees to go along
+      with it.  From the notes, it seemed like the participants
+      supported seeing the proposal tested.
+
+    - *Simplified commitments:* participants discussed the simplified
+      commitments protocol (see [Newsletter #120][news120 commitments]),
+      which is an idea to define which peer is responsible for proposing
+      the next change to the commitment transaction rather than allowing
+      either peer to propose a new commitment transaction at any time.
+      Putting one peer in charge eliminates the complexity of two
+      proposals being sent at roughly the same time, such as both Alice
+      and Bob wanting to add an [HTLC][topic htlc] simultaneously.  A
+      particular complication discussed in the notes were cases where
+      one of the peers didn't want to accept the other's peer's
+      proposal, which is a situation that's complicated to address in
+      the current protocol.  A downside of the simplified commitment
+      approach was that it may increase latency in some cases, as the
+      peer who is not currently responsible for proposing the next
+      change will need to request that privilege from its counterparty
+      before proceeding.  The notes did not indicate a clear resolution
+      to this discussion.
+
+    - *The specification process:* the participants discussed various
+      ideas for improving the specification process and the documents it
+      manages, including the current BOLTs and BLIPs plus other ideas
+      for documentation.  The discussion appeared highly varied and no
+      clear conclusions were apparent from the notes.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [How can I manually (on paper) calculate a Bitcoin public key from a private key?]({{bse}}118933)
+  Andrew Poelstra gives an overview of computation-by-hand verification
+  techniques, like [codex32][news239 codex32], before walking through how a
+  public key could be hand-derived from a private key, a process he estimates would
+  take at least 1500 hours, even with process optimizations.
+
+- [Why are there 17 native segwit versions?]({{bse}}118974)
+  Murch explains that [segwit][topic segwit] defined 17 values (0-16) for the [version bytes][bip341
+  witness program] field because of the existing availability of constant
```

Comment:
I think this should link to BIP141 rather. I would suggest either “witness version” or “version byte field”, but not “version bytes”:

```suggestion
  Murch explains that [segwit][topic segwit] defined 17 values (0-16) for the [witness version][bip141]
  because of the existing availability of constant
```

or: 

```suggestion
  Murch explains that [segwit][topic segwit] defined 17 values (0-16) for the [version byte][bip141]
  field because of the existing availability of constant
```

Reply:
Renamed the reference link. Changed 'version bytes' to 'witness version'